spec/cucumber/cli/configuration_spec.rb

```diff
@@ -5,11 +5,11 @@
 
 module Cucumber
   module Cli
-    describe Configuration do
-      module ExposesOptions
-        attr_reader :options
-      end
+    module ExposesOptions
```

Comment:
This now leaks into all potential other scenarios. Ideally we would write this as

```
let(:exposes_options) do
  Module.new do
    attr_reader :options
  end
end
```

Reply:
Actually, it was already leaking into all other scenarios.
So the fix of the offenses make it actually explicit.

So, I guess we could also thing of refactoring that as part of another PR

Reply:
Oki I'm happy to tackle it elsewhere, but lets try tackle this sooner rather than later as it's a reasonably large covering fix.

Reply:
Agree